### ISSUE TITLE
Is there any way to fine tune the ControlNet(using human pose) model using my own dataset?

### ISSUE THREAD

**Issue by wtqqtw** [lllyasviel/ControlNet#427]
I've followed the tutorial_train.py and the tutorial about how to train a ControlNet on huggingface.

I do not want to train a ControlNet conditioned on human pose, because I do not have that much data.

Is there any way to fine-tune your existing human pose-conditioned ControlNet model?

**Comment by geroldmeisinger** [comment]
I'm using the Huggingface diffusers control net training script https://huggingface.co/docs/diffusers/training/controlnet
it outputs checkpoints every 500 steps by default with all the intermediate data needed to resume the training

`
controlnet/
	config.json
	diffusion_pytorch_model.safetensors
optimizer.bin
random_states_0.pkl
scaler.pt
scheduler.bin
`

I don't know whether or not this is also the case for the original training script, or if this data is available for the original control nets. but this is what you would need.

**Comment by geroldmeisinger** [comment]
all duplicates on "resume training from official ControlNet"
https://github.com/lllyasviel/ControlNet/issues/233
https://github.com/lllyasviel/ControlNet/issues/337
https://github.com/lllyasviel/ControlNet/issues/411
https://github.com/lllyasviel/ControlNet/issues/427
